.github/ISSUE_TEMPLATE/antivirus.md

```diff
@@ -0,0 +1,48 @@
+
+* [Microsoft Defender](https://www.microsoft.com/wdsi/filesubmission) (previously called *Windows Defender*)
+* [MacAfee](https://www.microsoft.com/en-us/wdsi/filesubmission) (requires a paid *corporate* account to submit files)
+* [AVG](https://www.avg.com/en-us/false-positive-file-form)
+* [Qihoo-360](http://www.360totalsecurity.com/en/suspicion.html)
+* [Zillya](mailto:antivirus@zillya.com?subject=False%20Positive%20Submission&body=The%20sample%20is%20in%20a%20password%20protected%20zip%20file%0A%0AThe%20password%20for%20the%20attachment%20is%20infected) (via email only)
+
+
+### How do we know that our compiler chain (and therefore the bootloaders) really aren't infected?
+
+You can never know for sure if an executable is trustworthy but we have recompiled our bootloaders plenty of times on different machines giving bit for bit (or [as close as MSVC lets you get](https://bytepointer.com/articles/the_microsoft_rich_header.htm)) reproducibility. So unless they are all identically infected then this can't be it.
+
+On the other hand, we have seen through repeatedly submitting the same files for analysis at intervals that the results are far even from being consistent which puts their accuracy under serious question.
+
+If you're still not convinced then audit the [source code](https://github.com/pyinstaller/pyinstaller/tree/master/bootloader/src) and [build the bootloaders](https://pyinstaller.readthedocs.io/en/latest/bootloader-building.html) yourself.
+
+
+### What can you do to get it to run on your computer?
+
+* Disable your antivirus completely. This may sound radical but consider, it's clearly demonstrated that it doesn't work - what's the point in keeping it? You will also be amazed at how much faster your computer runs without it.
+* Replace your antivirus with a better one (Kaspersky or Avira to name a couple).
+
+
+### What can you do to get it working for users?
+
+* [Reporting false positives to AV vendors](#reporting-false-positives-to-av-vendors) is the correct way and some providers even provide APIs to do so automatically. They should eventually whitelist your software as OK. Bear in mind that users will likely need to first install some form of security update for that whitelist to propagate to their own computer before they stop seeing warnings.
+* [Recompiling the bootloaders](https://pyinstaller.readthedocs.io/en/latest/bootloader-building.html) yourself can make applications better accepted by the heuristics based AVs just because it makes yours different. In particular, using gcc instead of the MSVC compiler on Windows to completely through them off. Note that this is a double edged sword however - by being different your application will not benefit if PyInstaller's bootloaders get whitelisted. Also be aware that if enough people do this, the heuristics will swing the other way and you're back to square one again.
```

Comment:
"through them off" was probably supposed to be "throw them off"?

Reply:
uhm yeah that's what I meant...